# HG changeset patch
# User Richard Russon <rich@flatcap.org>
# Date 1464576126 -3600
#      Mon May 30 03:42:06 2016 +0100
# Branch gentoo-1.7
# Node ID cb73c0b019f1d20862ec0161c1ce1a3b575aa683
# Parent  7af6366e8f9d1881cc8546c65975e1abd60f687c
[PATCH 01/11] prefix-mailcap
From dd694e647dd2001c021ac0d8a6400bed04df2e7a Mon Sep 17 00:00:00 2001
# User Fabian Groffen <grobian@gentoo.org>
# Date 1279820983 -7200

Adjust mailcap search path for Prefix (offset installs).  Due to the
search path addition, should not cause any problems for non-offset
installs.
---
 Makefile.am | 2 +-
 init.c      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff -r 7af6366e8f9d -r cb73c0b019f1 Makefile.am
--- a/Makefile.am	Fri May 06 00:36:57 2016 +0100
+++ b/Makefile.am	Mon May 30 03:42:06 2016 +0100
@@ -46,7 +46,7 @@ mutt_DEPENDENCIES = $(MUTT_LIB_OBJECTS) 
 
 DEFS=-DPKGDATADIR=\"$(pkgdatadir)\" -DSYSCONFDIR=\"$(sysconfdir)\" \
 	-DBINDIR=\"$(bindir)\" -DMUTTLOCALEDIR=\"$(datadir)/locale\" \
-	-DHAVE_CONFIG_H=1
+	-DHAVE_CONFIG_H=1 -DPREFIX=\"$(prefix)/..\"
 
 AM_CPPFLAGS=-I. -I$(top_srcdir) $(IMAP_INCLUDES) $(GPGME_CFLAGS) -Iintl
 
diff -r 7af6366e8f9d -r cb73c0b019f1 init.c
--- a/init.c	Fri May 06 00:36:57 2016 +0100
+++ b/init.c	Mon May 30 03:42:06 2016 +0100
@@ -3356,7 +3356,7 @@ void mutt_init (int skip_sys_rc, LIST *c
   else
   {
     /* Default search path from RFC1524 */
-    MailcapPath = safe_strdup ("~/.mailcap:" PKGDATADIR "/mailcap:" SYSCONFDIR "/mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap");
+    MailcapPath = safe_strdup ("~/.mailcap:" PKGDATADIR "/mailcap:" SYSCONFDIR "/mailcap:" PREFIX "/etc/mailcap:/etc/mailcap:/usr/etc/mailcap:/usr/local/etc/mailcap");
   }
 
   Tempdir = safe_strdup ((p = getenv ("TMPDIR")) ? p : "/tmp");
